File: profiles/mo6tw/hud/savemenu.js
root.SaveMenu = {
    Type: SaveMenuType.MO6TW,
    SaveMenuBackgroundSprite: "SaveMenuBackground",
	EmptyThumbnailSprite: "EmptyThumbnail",
    QuickLoadTextSprite: "QuickLoadText",
    LoadTextSprite: "LoadText",
    SaveTextSprite: "SaveText",
    MenuTitleTextPos: { X: 62.0, Y: 33.0 },
    EntryStartX: 153,
    EntryXPadding: 512,
    EntryStartY: 102,
    EntryYPadding: 141,
    QuickLoadEntrySprite: "QuickLoadEntry",
    QuickLoadEntryHighlightedSprite: "QuickLoadEntryHighlighted",
    SaveEntrySprite: "SaveEntry",
    SaveEntryHighlightedSprite: "SaveEntryHighlighted",
    LoadEntrySprite: "LoadEntry",
    LoadEntryHighlightedSprite: "LoadEntryHighlighted",
    FadeInDuration: 0.2,
    FadeOutDuration: 0.2
};

root.Sprites["SaveMenuBackground"] = {
    Sheet: "Save",
    Bounds: { X: 0, Y: 0, Width: 1280, Height: 720 }
};

root.Sprites["EmptyThumbnail"] = {
    Sheet: "Save",
    Bounds: { X: 1281, Y: 1, Width: 178, Height: 108 }
};

root.Sprites["QuickLoadText"] = {
    Sheet: "Save",
    Bounds: { X: 1281, Y: 111, Width: 262, Height: 42 }
};
root.Sprites["SaveText"] = {
    Sheet: "Save",
    Bounds: { X: 1281, Y: 155, Width: 262, Height: 42 }
};
root.Sprites["LoadText"] = {
    Sheet: "Save",
    Bounds: { X: 1281, Y: 199, Width: 262, Height: 42 }
};

root.Sprites["QuickLoadEntry"] = {
    Sheet: "Save",
    Bounds: { X: 1, Y: 721, Width: 508, Height: 128 }
};
root.Sprites["SaveEntry"] = {
    Sheet: "Save",
    Bounds: { X: 511, Y: 721, Width: 508, Height: 128 }
};
root.Sprites["LoadEntry"] = {
    Sheet: "Save",
    Bounds: { X: 1021, Y: 721, Width: 508, Height: 128 }
};
root.Sprites["QuickLoadEntryHighlighted"] = {
    Sheet: "Save",
    Bounds: { X: 1, Y: 851, Width: 508, Height: 128 }
};
root.Sprites["SaveEntryHighlighted"] = {
    Sheet: "Save",
    Bounds: { X: 511, Y: 851, Width: 508, Height: 128 }
};
root.Sprites["LoadEntryHighlighted"] = {
    Sheet: "Save",
    Bounds: { X: 1021, Y: 851, Width: 508, Height: 128 }
};

File: profiles/mo8/hud/savemenu.lua
root.SaveMenu = {
    Type = SaveMenuType.MO8,
    DrawType = DrawComponentType.SystemMenu,
    SaveMenuBackgroundSprite = "SaveMenuBackground",
    EmptyThumbnailSprite = "EmptyThumbnail",
    QuickLoadTextSprite = "QuickLoadText",
    LoadTextSprite = "LoadText",
    SaveTextSprite = "SaveText",
    MenuTitleTextPos = { X = 60, Y = 30 },
    NextButtonSprite = "SaveMenuNextButton",
    NextButtonHighlightedSprite = "SaveMenuNextButtonHighlighted",
    NextButtonPosition = { X = 1586, Y = 972 },
    BackButtonSprite = "SaveMenuBackButton",
    BackButtonHighlightedSprite = "SaveMenuBackButtonHighlighted",
    BackButtonPosition = { X = 30, Y = 972 },
    BackButtonPositionVoice = { X = 808, Y = 972 },
    EntryStartX = 153,
    EntryXPadding = 512,
    EntryStartY = 102,
    EntryYPadding = 141,
    FadeInDuration = 0.2,
    FadeOutDuration = 0.2
};

root.Sprites["SaveMenuBackground"] = {
    Sheet = "Save",
    Bounds = { X = 0, Y = 0, Width = 1920, Height = 1080 }
};

root.Sprites["EmptyThumbnail"] = {
    Sheet = "Save",
    Bounds = { X = 1281, Y = 1, Width = 178, Height = 108 }
};

root.Sprites["QuickLoadText"] = {
    Sheet = "Save",
    Bounds = { X = 0, Y = 1848, Width = 438, Height = 90 }
};
root.Sprites["SaveText"] = {
    Sheet = "Save",
    Bounds = { X = 0, Y = 1652, Width = 438, Height = 90 }
};
root.Sprites["LoadText"] = {
    Sheet = "Save",
    Bounds = { X = 0, Y = 1750, Width = 438, Height = 90 }
};

root.Sprites["SaveMenuBackButton"] = {
    Sheet = "Data",
    Bounds = { X = 0, Y = 722, Width = 304, Height = 80 }
};

root.Sprites["SaveMenuBackButtonHighlighted"] = {
    Sheet = "Data",
    Bounds = { X = 0, Y = 898, Width = 304, Height = 80 }
};

root.Sprites["SaveMenuNextButton"] = {
    Sheet = "Data",
    Bounds = { X = 0, Y = 810, Width = 304, Height = 80 }
};

root.Sprites["SaveMenuNextButtonHighlighted"] = {
    Sheet = "Data",
    Bounds = { X = 0, Y = 634, Width = 304, Height = 80 }
};
